framework/db/Query.php

```diff
@@ -489,7 +487,7 @@ protected function queryScalar($selectExpression, $db)
             $this->limit = $limit;
             $this->offset = $offset;
 
-            if ($e !== null) {
+            if (isset($e)) {
```

Comment:
That's not a good practice overall to rely on abscense of a variable.

Reply:
@samdark yes, but this often used in current code
https://github.com/yiisoft/yii2/blob/6b1f70b883cc091644a30529d7d106c954abd1ce/framework/db/ActiveRelationTrait.php#L202 
https://github.com/yiisoft/yii2/blob/6b1f70b883cc091644a30529d7d106c954abd1ce/framework/db/ActiveRelationTrait.php#L272 
that's confused me... what's better: write code in "yii style" or based on good practices?